awslimitchecker/runner.py

```diff
@@ -358,6 +361,11 @@ def console_entry_point(self):
             ta_refresh_timeout=args.ta_refresh_timeout
         )
 
+        if args.skip_service is not None:
+            if args.service is not None:
+                raise SystemExit(0)
```

Comment:
This conditional needs a bit more work. If we're considering ``skip_service`` and ``service`` to be mutually exclusive, then that should be handled in ``parse_args``, by telling Argparse that the options are mutually exclusive (see [line 164](https://github.com/jantman/awslimitchecker/blob/master/awslimitchecker/runner.py#L164) for an example of adding a mutually exclusive argument group). 

Aside from that, on these lines we're raising a ``SystemExit`` with code 0, which implies success, but this is actually an error condition. That would be very confusing to a user, as the program would exit early with no indication of why.